### PR TITLE
Rename vfs representors

### PR DESCRIPTION
**Pull request by wmousa**
Rename vfs representors to be in style "<sriov_pf>_r_<vf_number>"

### CODE REVIEW

**Comment by wmousa** [comment]
@adrianchiris

File: switchdev_utils/config-service/switchdev_network_config [hunk 1]
```diff
@@ -266,7 +267,7 @@ def add_udev_rule_for_vf_representors(sriov_pf):
     udev_data_line = 'SUBSYSTEM=="net", ACTION=="add", ATTR{phys_switch_id}'\
                      '=="%s", ATTR{phys_port_name}=="pf%svf*", '\
                      'IMPORT{program}="%s $attr{phys_port_name}", '\
-                     'NAME="%s_$env{NUMBER}"' % (phys_switch_id,
+                     'NAME="%s_r_$env{NUMBER}"' % (phys_switch_id,
```
Comment (adrianchiris) [inline switchdev_utils/config-service/switchdev_network_config]: no need for underscore after `r`

**Review by adrianchiris — CHANGES_REQUESTED** [review]
please update commit message to reflect the fact that this PR also removes MOFED udev rule file for vf representors.

File: switchdev_utils/config-service/switchdev_network_config (hunk 1 above)
Comment (wmousa) [reply switchdev_utils/config-service/switchdev_network_config]: done

**Pull request merged**